scripts/rstats.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#!coding:utf-8
+
+# Copyright 2022 TiKV Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import re
+import json
+import argparse
+
+def main():
+    pattern = r'.*SlowLog.*'
+    slowstr = "SlowLog:"
+    grpc_pattern = "gRPC tikvpb.Tikv"
+    backoff_pattern = "backoff "
+
+    args = parse_args()
+    items = []
+    with open(args.slowlog, encoding = 'utf-8') as f:
+        for line in f.readlines():
+            matched = re.match(pattern, line, re.M|re.I)
+            if matched is not None:
+                log = json.loads(line[(line.index(slowstr) + len(slowstr)):])
```

Comment:
What if there is a line containing corrupted JSON?

Reply:
the script assumes that the slow log file is generated by tikv applications without manual modifications. If it's not a JSON, it means the file is mistakenly modified by someone, the script just not working since it breaks the assumption.